product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -48,6 +48,64 @@ Note: In this case, if the grace period is kept too high then writes will contin
 
 Having said that, most of the time, upon loss of the current Raft leader the new Raft leader gets elected by BDR within a few seconds provided more than half of the nodes (quorum) are still up. Hence, if the Raft leader is down but the write leader is still up then proxy can be configured to allow routing by keeping `consensus_grace_period` to a non-zero, positive value. The proxy will wait for the Raft leader to get elected during this period before stopping routing. This may be helpful in some cases where availability is more important.
 
+### Proxy health check
```

Comment:
Move this section to end of this page, under `Managing PGD Proxy`, parallel to [Proxy log location](https://www.enterprisedb.com/docs/pgd/latest/routing/proxy/#proxy-log-location). We don't want to highlight it and also probes are more about Managing Proxy.